docs/models.md

```diff
@@ -60,38 +75,76 @@ BoTorch provides several GPyTorch models to cover most standard BO use cases:
   which outcomes are modeled independently, given a list of any type of
   single-task GP. This model should be used when the same training data is not
   used for all outputs.
 
 ### Multi-Task GPs
-* [`MultiTaskGP`](../api/models.html#multitaskgp): a Hadamard multi-task,
-  multi-output GP using an ICM kernel, inferring the noise level (does not
+* [`MultiTaskGP`](../api/models.html#module-botorch.models.multitask): a Hadamard multi-task,
+  multi-output GP using an ICM kernel, inferring a homoskedastic noise level (does not
   require noise observations).
-* [`FixedNoiseMultiTaskGP`](../api/models.html#fixednoisemultitaskgp):
+* [`FixedNoiseMultiTaskGP`](../api/models.html#botorch.models.multitask.FixedNoiseMultiTaskGP):
   a Hadamard multi-task, multi-output GP using an ICM kernel, with fixed
   observation noise levels (requires noise observations).
+* [`KroneckerMultiTaskGP`](../api/models.html#botorch.models.multitask.KroneckerMultiTaskGP): A multi-task,
+  multi-output GP using an ICM kernel, with Kronecker structure. Useful for
+  multi-fidelity optimization.
 
 All of the above models use Matérn 5/2 kernels with Automatic Relevance
 Discovery (ARD), and have reasonable priors on hyperparameters that make them
 work well in settings where the **input features are normalized to the unit
 cube** and the **observations are standardized** (zero mean, unit variance).
 
+## Other useful models
+
+* [`ModelList`](../api/models.html#botorch.models.model.ModelList): a multi-output model in which outcomes
+  are modeled independently, as in `ModelListGP`, but the component models do not all
+  need to be GPyTorch models.
+* [`SingleTaskMultiFidelityGP`](../api/models.html#botorch.models.gp_regression_fidelity.SingleTaskMultiFidelityGP) and
+  [`FixedNoiseMultiFidelityGP`](../api/models.html#botorch.models.gp_regression_fidelity.FixedNoiseMultiFidelityGP):
+  Models for multi-fidelity optimization.  For more on Multi-Fidelity BO, see the
+  [tutorial](../tutorials/discrete_multi_fidelity_bo).
+* [`HigherOrderGP`](../api/models.html#botorch.models.higher_order_gp.HigherOrderGP): A GP model with
+  matrix-valued predictions, such as images or grids of images.
+* [`PairwiseGP`](../api/models.html#module-botorch.models.pairwise_gp): A probit-likelihood GP that
+  learns via pairwise comparison data, useful for preference learning.
+* [`ApproximateGPyTorchModel`](../api/models.html#botorch.models.approximate_gp.ApproximateGPyTorchModel): for
+  efficient computation when data is large or responses are non-Gaussian.
+* [Deterministic models](../api/models.html#module-botorch.models.deterministic), such as
+  [`AffineDeterministicModel`](../api/models.html#botorch.models.deterministic.AffineDeterministicModel),
+  [`AffineFidelityCostModel`](../api/models.html#botorch.models.cost.AffineFidelityCostModel),
+  [`FixedSingleSampleModel`](../api/models.html#botorch.models.deterministic.FixedSingleSampleModel),
```

Comment:
Not sure we need FixedSingleSampleModel here, this one is kind of special and a bit of a hack.

Reply:
sounds good, also it's not in `models/__init__.py`